### PR TITLE
Enable DLM module for intel NUC

### PR DESCRIPTION
**Pull request by telphan**
Connects to resin-os/resinos#246

### CODE REVIEW

File: layers/meta-resin-intel/recipes-kernel/linux/linux-yocto_%.bbappend [hunk 1]
```diff
@@ -7,3 +7,15 @@ RESIN_CONFIGS_append_intel-corei7-64 = " nvme"
...
     CONFIG_BLK_DEV_NVME=y \
     "
+
+#
+# Support for DLM module
+#
+RESIN_CONFIGS_append_intel-corei7-64 = " dlm"
+RESIN_CONFIGS_DEPS[dlm] = " \
+    CONFIG_SYSFS=y \
+    CONFIG_CONFIGFS_FS=y \
```
Comment (floion) [inline layers/meta-resin-intel/recipes-kernel/linux/linux-yocto_%.bbappend]: Why did you add this as built-in?

Comment (telphan) [inline layers/meta-resin-intel/recipes-kernel/linux/linux-yocto_%.bbappend]: I thought that dependencies on modules need to be built in. Now I see I am mistaken.

**Comment by willnewton** [comment]
DLM is generic functionality, why don't we enable it for all architectures?

**Comment by agherzan** [comment]
It is @willnewton but for now we decided to only include it in intel builds. In the future we may roll them to other devices as per user requests.
